docs/manual/docs/cmd/spfx/project/project-rename.md

```diff
@@ -0,0 +1,39 @@
+# spfx project rename
+
+Renames SharePoint Framework project
```

Comment:
Doesn't match description from the command `Rename SharePoint Framework project` (rename instead of renames)

Reply:
In the issue #1349 description is mentioned as `Renames SharePoint Framework project`. Please confirm if this should be updated to `Rename SharePoint Framework project`

Reply:
Got it now. Updated all references of `Rename SharePoint Framework project` to `Renames SharePoint Framework project` to match the issue description.